crates/services/sync/src/ports.rs

```diff
@@ -34,8 +34,8 @@ pub enum PeerReportReason {
     InvalidTransactions,
 }
 
-#[cfg_attr(any(test, feature = "benchmarking"), mockall::automock)]
 #[async_trait::async_trait]
+#[cfg_attr(any(test, feature = "benchmarking"), mockall::automock)]
```

Comment:
Yeah it's realted to the PR even if it's not very clear. It's because previously as the code was synchronous the request wasn't in parallel and so it was impossible to test if some work happens while answers from another peer hasn't arrive yet (and end-up with incorrect data but because other job has been done in the same time we have cached data). Now we can sleep during a p2p call : https://github.com/FuelLabs/fuel-core/blob/d9ef69ea61d1f8fe1647092e23ed135cb6109404/crates/services/sync/src/import/tests.rs#L391 that stimulate someone taking a bit more time to answers, the correct one answers directly and after the failed one answers and we re-ask the correct one has been cached.